vispy/testing/_testing.py

```diff
@@ -242,6 +242,30 @@ def requires_img_lib():
     return np.testing.dec.skipif(not has_img_lib, 'imageio or PIL required')
 
 
+def has_ipython(version='3.0'):
+    """function that checks the presence of IPython"""
+    try:
+        import IPython  # noqa
+    except Exception:
+        return False, "IPython library not found"
+    else:
+        if LooseVersion(IPython.__version__) >= LooseVersion(version):
+            return True, "IPython present"
+        else:
+            message = (
+                "current IPython version: (%s) is "
+                "less than expected version: (%s)") % \
```

Comment:
I mean say "older than required version" instead of "less than expected version"